docs/source/tutorial-pyfunc.rst

```diff
@@ -0,0 +1,368 @@
+
+The first thing that ``store_pyfunc_model`` does is confirm that the MLflow Tracking Server is running on a URI that starts with ``http`` or ``https``, as discussed in :ref:`setup`.
+
+The code then uses the ``joblib`` library to serialize the model to disk. Using serialization/deserialization to save and load the model and related files can make for clear and efficient code, but it is not required. You can store any type of file in the ``artifacts`` directory: JSON files, binary weight files, vector database files, and so forth. 
+
+The ``artifacts`` dictionary is then created. Its ``classifier`` key is set to the name of the file containing the serialized model: ``"iris_classifier.joblib"``.
+
+An instance of the ``IrisClassifier`` class is then instantiated. This class derives from :py:class:`mlflow.pyfunc.PythonModel`. As discussed at the beginning of this section, this class implements the ``load_context`` and ``predict`` methods. The ``IrisClassifier`` implementation of those methods will be discussed in the next section. 
+
+After the ``IrisClassifier`` instance is created, the ``mlflow.pyfunc.log_model`` method is called. This method takes the following arguments:
+
+- ``python_model``: The instance of the ``IrisClassifier`` class we just created.
+- ``artifacts``: The dictionary containing the ``"classifier"`` key and the ``"iris_classifier.joblib"`` value.
+- ``artifact_path``: The path to the artifacts directory within the root directory of the run within the artifacts repository of the MLflow Tracking Server.
+- ``signature``: The signature of the model, as inferred in the ``initialize_and_train`` function.
+
+For more optional arguments, see :py:func:`mlflow.pyfunc.log_model`'s API documentation.
+
+The ``mlflow.pyfunc.log_model`` method returns a ``ModelInfo`` object. This object contains the ``run_id`` of the run that logged the model. This ``run_id`` is returned by the ``store_pyfunc_model`` function and is used in the next section.
+
+Review and register the stored ``pyfunc`` model in the MLflow UI
+----------------------------------------------------------------
+
+Open a Web browser and go to the address at which you are running the MLflow Tracking Server (for instance, ``http://localhost:5000``). You should see your just-logged run. Click on the run to see the run's details. You should see something like:
+
+.. image:: _static/images/tutorial-pyfunc/run-details.png
+    :alt: Screenshot of the Run Details page of the successfully run Iris Classifier in the MLflow UI
+
+Here, you can see the Run ID (1), the accuracy of the trained model (2), and the artifacts resulting from your call to ``log_model``. In addition to the **iris_classifier.joblib** you specified, there are all the files necessary to recreate the runtime environment (**conda.yaml**, **requirements.txt**, etc.) and the infrastructure for loading your custom model (**MLmodel** and **python_model.pkl**).
```

Comment:
perhaps instead of "infrastructure" we just say "the metadata and files required to load your custom model"